vendor/github.com/rh-ecosystem-edge/eco-goinfra/pkg/sriov/policy.go

```diff
@@ -1,6 +1,7 @@
 package sriov
 
 import (
+	"context"
```

Comment:
All the ecogoinfra changes should be done in ecoinfra project directly by openning PR. And After that bump version here to use it.